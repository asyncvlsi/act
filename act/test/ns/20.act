

namespace test {

pint x;

defproc ckt(bool x)
{
  prs {
   x -> x-
  }
}

ckt c;

}
