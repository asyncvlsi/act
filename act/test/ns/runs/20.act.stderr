ERROR: File `20.act', line 14, col 5
	Cannot instantiate processes within a namespace
